python/packages/autogen-agentchat/src/autogen_agentchat/agents/_assistant_agent.py

```diff
@@ -66,6 +67,8 @@ class AssistantAgent(BaseChatAgent):
             If a handoff is a string, it should represent the target agent's name.
         description (str, optional): The description of the agent.
         system_message (str, optional): The system message for the model.
+        max_tool_iterations (int, optional): The maximum number of tool iterations to run before returning the response.
+        return_only_response (bool, optional): If `True`, the agent will only return the final response and not the tool calls.
```

Comment:
Perhaps `reflect_on_tool_call`? I think using more action-centric naming is better in this case to describe what is happening.

Reply:
for which one?
I removed return_only_response